### ISSUE TITLE
Support for RabbitMQ Clusters

### ISSUE THREAD

**Issue by MarcDufresne** [veegee/amqpy#33]
Hi, I recently stumbled upon this library and replaced my kombu setup with it. So far it works great and is faster than kombu. 

Now the only problem I'm having is that amqpy doesn't seem to support the amqp connection string (e.g.: `amqp://<user>:<pass>@<host>:<port>`) when establishing a connection. Moreover, it looks like there is no way of connecting to multiple servers in a cluster. The way I do it with kombu is by simply providing `amqp://<user>:<pass>@<host1>:<port1>;amqp://<user>:<pass>@<host2>:<port2>;amqp://<user>:<pass>@<host3>:<port3>` as the connection parameter. 

Maybe I missed something in the code, you can tell me if I'm wrong. It's the only thing right now keeping me from using this library for production purpose, which is a shame considering how well it works.

**Comment by veegee** [comment]
You're right, these two features are not supported. I would like to add support for this, but having difficulty finding the time for it at the moment.

The connection string is easy enough to implement. As far as the clusters go, I'll need to do a bit of research. I was also in the middle of redoing the underlying concurrency mechanism.

**Comment by MarcDufresne** [comment]
I just opened a pull request #34 to add support for the AMQP connection string, tell me what you think about it.
